rfcs/react-components/convergence/semantic-shared-colors.md

```diff
@@ -0,0 +1,195 @@
+| `green`      | success               | Replace by semantic `success`.                                                                                                      |
+| `darkOrange` | severe                | **severe** should not be used anywhere. Keep `darkOrange` (temporarily) for Teams compatibility.                                    |
+| `yellow`     | warning               | Replace by semantic `warning`, mapping changed from `yellow` to `orange`                                                            |
+| `berry`      | OOF                   | No change                                                                                                                           |
+| `lightGreen` | available             | No change                                                                                                                           |
+| `marigold`   | away, persona         | No change                                                                                                                           |
+
+Note: even though the task now is to introduce the semantic names only for the **status** tokens, in the future we might be asked to do the same change for **presence** tokens.
+
+### Other platforms
+
+I briefly checked react native packages for iOS and Android produced by token pipeline, they use tokens like `warningBackground1`.
+
+## Detailed Design or Proposal
+
+There are multiple options how to address the problem. This chapter describes the options and their pros and cons.
+
+### 1. Introduce new semantic status tokens
+
+Add 3 new semantic colors:
+
+1. danger
+2. success
+3. warning
+
+For each of the 3 new colors, add 9 (or 10) color slots (see the table above).
+
+Update the sources in the library to use the semantic color names instead of the current shared color names.
+
+Find a way how to get rid of the legacy tokens which are no longer needed:
+
+- `green` (10 tokens)
+- `yellow` (10 tokens)
+
+Once the legacy tokens are not used by the library code, there are three possible ways how to get rid of the tokens:
+
+#### 1A: "deprecate" the tokens and remove them in the next major release
+
+#### 1B: "deprecate" the tokens and create a V2 theme which does not contain them - let applications switch when they are ready
+
+#### 1C: big bang - break semver, remove the tokens immediately
+
+#### Pros and Cons
+
+- 👍 Decouples the persona and status shared colors, moves us to architecturally cleaner solution.
+- 👍 Designers can "freely" change the status colors' mapping.
+- 👍 Partners can override persona and status colors independently.
+- 👎 We will remove some "basic" colors `green`, `yellow` - are we sure those have been used as semantic colors only?
+  approach
+- 👎 1A: adds ~30 new tokens
+- 👎 If design decides to do the same for **presence** tokens, we will need to add another ~40 tokens to follow the same
```

Comment:
And I'm sure there will be new families of tokens requested for scenarios we don't forsee today.